src/Pages/SignUp/SignUpInput.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import styled from "styled-components";
+
+const SignUpInput = props => {
+  const { name, title, type, validation, inputAlert, setName } = props.data;
+  return (
+    <InputContainer>
+      <PlaceHolder>{title}</PlaceHolder>
+      <Input name={name} type={type} onChange={e => setName(e.target.value)} />
+      {validation && <InputAlert>{inputAlert}</InputAlert>}
+    </InputContainer>
+  );
+};
+
+const InputContainer = styled.div`
+  display: block;
```

Comment:
```suggestion
const InputContainer = styled.div`
```
div 태그는 이미 block 요소이기 때문에 안넣어주셔도 될거 같아요

Reply:
넵! 반영했습니다 👍🏻